Editor/PluginSettings/SettingsWindow.cs

```diff
@@ -178,9 +175,6 @@ private void DrawActiveTab()
                         case SettingsState.TabSdk:
                             DrawSdkTab();
                             break;
-                        case SettingsState.TabTest:
-                            DrawTestTab();
-                            break;
```

Comment:
Same question as above: can you clarify what this corresponded to? I guess i'm not familiar with the UX and mocks which is why i have some trivial questions!